.eslintrc.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "indent": [
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "single"
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+        "comma-dangle": [
+            "error",
+            "never",
+        ],
+        "no-console": [
+            "error",
+        ],
+        "max-len": [
+            "error",
+            80, 
```

Comment:
You can also add in a value for the indentation for the rule to account for. This works fine, however, so it's just an fyi.